api/services/snapshots/v1/snapshots.proto

```diff
@@ -114,17 +114,19 @@ message Info {
 	string name = 1;
 	string parent = 2;
 	Kind kind = 3;
+	string snapshot_id = 4;
```

Comment:
please have the added fields at the end of the proto so that we have fewer compatibility issues